### PR TITLE
fixes phazon construction message

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
fix: phazon construction message is fixed
/:cl:

### CODE REVIEW

File: code/game/mecha/mecha_construction_paths.dm [hunk 1]
```diff
@@ -1606,7 +1606,7 @@
 				user.visible_message("[user] unfastens the scanner module.", "<span class='notice'>You unfasten the scanner module.</span>")
 		if(14)
 			if(diff==FORWARD)
-				user.visible_message("[user] secures [I].", "<span class='notice'>You secure [I].</span>")
+				user.visible_message("[user] secures the capacitor.", "<span class='notice'>You secure the capacitor.</span>")
```
Comment (ninjanomnom) [inline code/game/mecha/mecha_construction_paths.dm]: What is this fixing? This should be fixed so `I` is still used

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/game/mecha/mecha_construction_paths.dm (hunk 1 above)
Comment (Tlaltecuhtli) [reply code/game/mecha/mecha_construction_paths.dm]: [I] is the screwdriver, the other mechs use this format

Comment (ninjanomnom) [reply code/game/mecha/mecha_construction_paths.dm]: Yes I know, I told others in the past to use this format. Why are you changing away from it?

Comment (ShizCalev) [reply code/game/mecha/mecha_construction_paths.dm]: A capacitor was already installed to the mech in the previous step. This is the proceeding step where you secure it using a screwdriver. This is correct, unless literally the rest of this entire file is wrong.

Comment (ninjanomnom) [reply code/game/mecha/mecha_construction_paths.dm]: Oh godamit I was misreading where this was

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**